blank-canvas-blocks/sass/base/_normalize.scss

```diff
@@ -16,6 +16,10 @@ img {
 	max-width: 100%;
 }
 
+hr {
+	border: none; // Undo the wp-admin common.css: https://github.com/WordPress/WordPress/blob/fc6e4cbe3ef1dcd3a9977272d7ddcc413acdde0d/wp-admin/css/common.css#L865-L869
```

Comment:
[This PR ](https://github.com/WordPress/gutenberg/pull/30235)should undo what wp admin defines, but the browser defaults are not `border: none;` so it will be different if/when that PR lands. `add_theme_support( 'wp-block-styles' );` does provide some defaults for the separator, though.

Reply:
That PR just landed, by the way